Integrations/MicrosoftGraphFiles/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [Unreleased]
+
+test test playbook
```

Comment:
```suggestion
 - 
```
no need for release notes if you just change/add tests and a mistake in the docs.